src/strategies/views/speakers-tvs-view.test.ts

```
@@ -0,0 +1,9 @@
+import { describe, expect, test } from 'vitest';
+import { SpeakersTvsViewStrategy } from './speakers-tvs-view';
+
+describe('SpeakersTvsViewStrategy.generate', () => {
+  test('returns empty speakers & TVs view', async () => {
+    const view = await SpeakersTvsViewStrategy.generate({}, {} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)